src/Entity/User.php

```diff
@@ -12,23 +12,14 @@
 use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
 use Symfony\Component\Security\Core\User\UserInterface;
 
+
 #[ORM\Entity(repositoryClass: UserRepository::class)]
 #[ORM\HasLifecycleCallbacks]
 #[Gedmo\SoftDeleteable(fieldName: 'deletedAt', timeAware: false, hardDelete: true)]
 #[ORM\Table(name: '`user`')]
 class User implements UserInterface, PasswordAuthenticatedUserInterface
 {
     use SoftDeleteableEntity;
-   
-    public const ROLE_ADMIN = 'ROLE_ADMIN';
-    public const ROLE_HOST = 'ROLE_HOST';
-    public const ROLE_EXPERIENCER = 'ROLE_EXPERIENCER';
-
-    public const ROLE_HIERARCHY = [
-        self::ROLE_EXPERIENCER => [self::ROLE_EXPERIENCER],
-        self::ROLE_HOST => [self::ROLE_HOST, self::ROLE_EXPERIENCER],
-        self::ROLE_ADMIN => [self::ROLE_ADMIN, self::ROLE_HOST, self::ROLE_EXPERIENCER]
-    ];
 
```

Comment:
Please be sure to use the latest version of User entity as some constants needed for auth are missing in this branch.